feature: datasource for virtual machines [issue 196] (#498)
hashicorp/packer-plugin-vsphere: docs/README.md

  This builder deploys and publishes new virtual machine to a vSphere Supervisor cluster using VM
  Service.

#### Data Sources

- [vsphere-virtual_machine](/packer/integrations/hashicorp/vsphere/latest/components/data-source/vsphere-virtual_machine) -

>   This datasource returns name of existing virtual machine that matches all defined filters to use

Reviewer: ```suggestion
  This data source returns the name of a virtual machine that matches all defined filters.
```
